api/integrations/trac/zulip_trac.py

```diff
@@ -36,6 +36,7 @@
 
 from trac.core import Component, implements
 from trac.ticket import ITicketChangeListener
+from typing import Any, Text
```

Comment:
This should be inside an `if False`, so that this script doesn't require the `typing` module to be present